.gitignore

```diff
@@ -10,3 +10,4 @@ esm
 .vscode
 examples/**/yarn.lock
 package-lock.json
+playground/**
```

Comment:
Oh I think it's not included in the PR?

Reply:
```yarn prepare:vite``` and ```yarn prepare:next``` will use git to get playground templates from [vite-swr](https://github.com/promer94/vite-swr.git) and [next-swr](https://github.com/promer94/next-swr).

So developers will not accidentally commit their changes to the playground